tests/FSharpPlus.Tests/Validations.fs

```diff
@@ -11,7 +11,27 @@ module Validation =
   open FSharpPlus.Data
   open Validation
   open FSharpPlus.Tests.Helpers
-
+  
+  let private isSuccess =
+    function
+    | Success _ -> true
+    | Failure _ -> false
+  
+  let private isFailure =
+    function
+    | Success _ -> false
+    | Failure _ -> true
+    
+  let private getSuccess =
+    function
+    | Success s -> s
+    | Failure _ -> failwith "It's a failure"
```

Comment:
We have `defaultValue` and `defaultWith`, same as in `Result` which now are part of FSharp.Core.

I think having a function throwing a fixed exception, is not worth, we can just use the above functions in a more explicit way. It's also more predictable from the user perspective.

Reply:
both already exist for the Validation type